scripts/systemd/nut-monitor.service.in

```diff
@@ -17,7 +17,8 @@ PartOf=nut.target
 
 [Service]
 EnvironmentFile=-@CONFPATH@/nut.conf
-ExecStart=@SBINDIR@/upsmon
+ExecStart=@SBINDIR@/upsmon -F
+ExecReload=@SBINDIR@/upsmon -c reload
 PIDFile=@PIDPATH@/upsmon.pid
 Type=forking
```

Comment:
`Type = forking` needs to go.

Reply:
Good point, now I wonder how it is in "competing" PRs. We have several on this subject, some from myself, and some for different components, so I intend to juggle the existing proposals into a consistent solution with things named and behaving the same in different daemons we have in NUT.

Reply:
Reflected in #683 follow-up